dataModel.OSLO.PassengerTransportHubs/BicycleParkingStationPTH-AP/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
This relationship is not a property?  I cannot see the link to the entity related to it.